var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -111,6 +111,11 @@ def _ld_library_path(self):
         for dir in dirs:
             yield join_path(self.component_path, 'linux', dir)
 
+    @property
```

Comment:
Why does bootstrapped %oneapi need this, but bootstrapped %intel does not?

Reply:
We shouldn't, because the `-classic` compiler installs into the bin directory, not into a subdirectory like `intel-oneapi-compilers`.